src/main/java/dk/sdu/se_f22/searchmodule/twowaysynonyms/DatabaseOperator.java

```diff
@@ -5,11 +5,12 @@
 import java.util.UUID;
 
 public interface DatabaseOperator extends Filterable {
-    UUID create(String synonym);
-    UUID create(String synonym, String groupMember);
-    ResultSet read(String synonym);
-    UUID updateGroupID(String synonym, String relatedSynonym);
-    UUID updateSpelling(String synonym, String correctedSpelling);
+    String create(String synonym);
+    String create(String synonym, String groupMember);
+    Synonym read(String synonym);
+    ArrayList<Synonym> readAll(String synonym);
```

Comment:
I don't get the reason why there is a need to create a new method here, couldn't the old one _read()_ have been used?

Reply:
I believe that the method is there to dump the whole synonym table if needed